documentation/index.md

```diff
@@ -8,7 +8,7 @@ title: Appyx – Overview
 ![badge-jvm](https://img.shields.io/badge/platform-jvm-orange)
 ![badge-macos](https://img.shields.io/badge/platform-macos-purple)
 ![badge-js](https://img.shields.io/badge/platform-js-yellow)
-![badge-ios](https://img.shields.io/badge/platform-🔄ios🔄-lightgray)
+![badge-ios](https://img.shields.io/badge/platform-ios-lightgray)
```

Comment:
since we not have iOS target, let's add a guide tp set up the environment correctly https://github.com/JetBrains/compose-multiplatform-template#set-up-the-environment

Reply:
Sounds great! Should I create a separate task for this? As we should be adding the setup for other platforms as well.